src/.github/workflows/pre-commit.yml.jinja

```diff
@@ -15,19 +15,22 @@ jobs:
     runs-on: ubuntu-20.04
 {%- else %}
     runs-on: ubuntu-22.04
+{%- endif %}
+{%- if odoo_version < 11 %}
+    container:
+      image: python:2.7.18-buster
```

Comment:
This may not be necessary since the ubunu-20.04 runner has python2 preinstalled.